### DISCUSSION TITLE
Will this app work on other DEs besides GNOME?

### DISCUSSION THREAD

**Discussion by trymeouteh** [andyholmes/valent#746] [General]
Will this app work on other desktop environments besides GNOME?

**Comment by Vortigern-The-Grey** [comment]
Would love to know this, on hyprland and would use this instead of KDE connect bc qt theming is a pai

**Comment by andyholmes** [comment]
From the [FAQ](https://github.com/andyholmes/valent/discussions/163):

> **Q: Valent for non-GNOME environments?**
>
> The core of Valent is written in C with the GNOME platform libraries and does not (necessarily) depend on a GUI toolkit. Most functionality is implemented in plugins, including the gnome plugin which provides the default GTK4/Adwaita UI.
>
> An alternate UI plugin (plain GTK4, Granite, TUI, etc) may be accepted into the main project, if it is well written and comes with a reasonable commitment to maintain it. If you have plans to contribute such a plugin, please reach out by email or in discussions.

If your desktop support Portals and standard freedesktop.org interfaces, then it should work. But nothing in Valent is particular supported at this point :slightly_smiling_face:

**Comment by zachvlat** [comment]
It works also for KDE Plasma 6. The only issue is that it "randomly" closes. I 'm assuming this is done when it's not being used. If I 'm continuously using my PC it is not killing it self.
